builtin/credential/ldap/path_config.go

```diff
@@ -208,6 +215,10 @@ func (b *backend) pathConfigWrite(ctx context.Context, req *logical.Request, d *
 		}, nil
 	}
 
+	if err := logical.EndTxStorage(ctx, req); err != nil {
```

Comment:
@suprjinx One bug: this needs to go above the `checkConfigUserFilter` call; those are simply warnings, not fatal errors and the transaction should still be committed rather than rolled back by the defer above.